app/feature/feature-home/src/main/kotlin/com/hedvig/android/feature/home/home/navigation/HomeGraph.kt

```diff
@@ -47,6 +49,9 @@ fun NavGraphBuilder.homeGraph(
         navigateToHelpCenter = { navigateToHelpCenter(backStackEntry) },
         openUrl = openUrl,
         openAppSettings = openAppSettings,
+        openChat = {
+          openChat(backStackEntry, null)
```

Comment:
If `null` is always passed here regardless, perhaps we can just not ask for the context and let it be null which is the default param further up where we do the actual navigation call

Reply:
Actually I saw this afterwards, we already pass `onStartChat` here ~8 parameters before this one, so you can completely remove this one instead.

Reply:
Yep, missed it completely :)